velox/dwio/type/fbhive/HiveTypeParser.cpp

```diff
@@ -75,8 +75,12 @@ std::shared_ptr<const Type> HiveTypeParser::parse(const std::string& ser) {
   remaining_ = folly::StringPiece(ser);
   Result result = parseType();
   VELOX_CHECK(
-      !(remaining_.size() != 0 && (TokenType::EndOfStream != lookAhead())),
-      "Input remaining after type parsing");
+      remaining_.size() == 0 || TokenType::EndOfStream == lookAhead(),
+      fmt::format(
```

Comment:
`format` is automatically applied in `VELOX_CHECK`, so just append the format string and extra arguments directly to the `VELOX_CHECK` call